sdks/java/core/src/main/java/org/apache/beam/sdk/schemas/transforms/Group.java

```diff
@@ -364,19 +365,43 @@
     }
   }
 
+  public abstract static class AggregateCombiner<InputT>
```

Comment:
Add a javadoc comment for this interface and its methods.

Reply:
Added